examples/certs/ed25519/README.md

```diff
@@ -0,0 +1,12 @@
+# Example keys
```

Comment:
I don't remember if I commented on this before (cannot find it now, at least), but why are these under the existing `certs` folder? I would expect a `keys` folder to be more appropriate?